app/src/main/res/values/strings.xml

```diff
@@ -1,16 +1,16 @@
 <resources>
-    <string name="app_name">MyPokedex</string>
-    <string name="hint_pokemon_search">Search Pokemon…</string>
-    <string name="description_profile_buttton">Profile Button</string>
-    <string name="description_camera_button">Camera button</string>
-    <string name="description_save_button">Save Button</string>
-    <string name="description_folder_button">Folder button</string>
-    <string name="description_location_image">Pokemon Location Image</string>
-    <string name="hint_pokemon_description">Enter description…</string>
-    <string name="label_pokemon_description">Pokemon Description</string>
-    <string name="label_pokemon_name">Pokemon Name</string>
-    <string name="label_pokemon_type">Pokemon Type</string>
-    <string name="hint_pokemon_name">Enter name…</string>
-    <string name="hint_pokemon_type">Enter type…</string>
-    <string name="label_pokemon_sightings">Pokemon Sighting Locations</string>
+    <string name = "app_name">MyPokedex</string>
```

Comment:
changing the space between = signs as in [Kotlin coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html#formatting)